README.md

```diff
@@ -1,33 +1,29 @@
-# Копия итогового проекта с курса "промышленное программирование" [школы программистов](https://informatics.ru)
----
+# The final project for the "Industrial Programming" course of [MSHP](https://informatics.ru)
```

Comment:
I would make a separate Russian translation of the readme and move this disclaimer there. Nobody outside Moscow and its districts needs this disclaimer. Mention that this is a training project is already written below.